app/controllers/graphql_explorer_controller.rb

```diff
@@ -0,0 +1,12 @@
+class GraphQLExplorerController < ApplicationController
+  before_action :check_privileges
+
+  after_action :cleanup_action
+
+  menu_section :graphql
+
+  def index
+    @in_a_form = true
+    @layout = "graphql_explorer"
```

Comment:
The menu item id is `graphql`, not `graphql_exporer`... they need to match for the menu item to be active